Week3/homework/step4-bonus.js

```diff
@@ -3,8 +3,24 @@
 const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
 
 // Add your function here. Try and come up with a good name for this function
-
+function uniqueElement(myArr) {
+    const els = myArr.map(el => el);
```

Comment:
I was trying to maybe improve this function a little more well it needs think, but this line right here is just creating a new array by `.map` which is not it's work, you can create a one simply by `myArr.slice()` or `[...myArr]`